frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -289,86 +301,112 @@ const Map: React.FC<MapProps> = ({
     }
   };
 
+  const [infoOpen, setInfoOpen] = React.useState(false);
```

Comment:
could put these in the context as well to keep the impact on the map code small.